### DISCUSSION TITLE
Sources for the genre "Conversation"

### DISCUSSION THREAD

**Discussion by ximenina** [MorphDiv/TeDDi_sample#276] [Ideas]
Our starting point for face-to-face conversations is CHILDES. However, other talkbanks are available on the [website ](https://sla.talkbank.org/]), where CHILDES is hosted.

-  Which talkbanks are the most appropriate to be included in TeDDi?

- Do you know some other sources where we can extract text for this genre that covers many languages, included low-resourced languages?

@tsamardzic @christianbentz @bambooforest

**Comment by ximenina** [comment]
From [talkbank.org](https://www.talkbank.org/) I think the following corpora is suitable:

Conversation Banks

[CABank](https://ca.talkbank.org)

[SamtaleBank](https://samtalebank.talkbank.org/)

[ClassBank](https://class.talkbank.org)

Child Language Banks

[CHILDES](https://childes.talkbank.org/)

[PhonBank](https://phonbank.talkbank.org/)

[HomeBank](https://homebank.talkbank.org/)

* We just have to check the annotation specifications to be able to extract the text and filter out unwanted tags and annotations (as much as possible).

**Comment by ximenina** [comment]
[Spoken Corpora from the Clarin project](https://www.clarin.eu/resource-families/spoken-corpora)

It comprises several sources with  transcriptions of spontaneous and planned speech. 

It covers 15 languages: Arabic, Czech, Dutch, Estonian, Finnish, French, German, Hungarian, Italian, Nepali, Norwegian, Polish, Skoti Saami, Slovenian, Spanish, and Swedish.

 - We just need to check that the resource is downloable, that the genre is suitable (for example, spontaneous speech, interviews, conversations, etc) and be aware of the particular tags, anotations used in each corpus.

**Comment by ximenina** [comment]
Other potential resources:

- [OpenSLR](https://www.openslr.org/) is a site that hosts speech and language resources. Some of them might be useful especially when they have orthographic transcription available (and the corpus corresponds to the Conversation genre).  So far, I've identified these resources:

 [Multilingual TEDx](https://www.openslr.org/100/) 
 [MAGICDATA Mandarin Chinese Conversational Speech Corpus](https://www.openslr.org/123/) 


- There are many sources of spoken corpora. Unfortunately, they do not properly represent conversational corpora, i.e., they come from Wikipedia or predefined sentences that the speakers read. In any case, some of these multilingual sources might be useful to obtain text for the under-resource languages (although different genre):

[CommonVoice](https://commonvoice.mozilla.org/) Hundreds of languages, many low-resourced

https://www.openslr.org/79/ Kannada critically endangered 
https://www.openslr.org/126/ Kannada critically endangered

**Comment by ximenina** [comment]
* I updated the potential sources for conversational corpora, in case you want to have a look. @vukbatanovic @tsamardzic @christianbentz

**Comment by tsamardzic** [comment]
There might be some data for us here too: https://www.amazon.science/blog/amazon-releases-51-language-dataset-for-language-understanding (might not be for conversation)
